src/app/Plans/components/PlanDetails.tsx

```diff
@@ -48,6 +50,13 @@ const PlanDetails: React.FunctionComponent<IPlanDetailsProps> = ({ plan }: IPlan
     ? warmCriticalConcerns.filter((label) => someVMHasConcern(selectedVMs as IVMwareVM[], label))
     : [];
 
+  const hooks = useHooksQuery();
+  const getHookDefinition = (name) => {
```

Comment:
I'd either call this something like `isPlaybookHook` or have it return `'playbook' | 'image'` explicitly. `getHookDefinition` doesn't indicate what the boolean returned means. Just a nitpick though, it doesn't really matter.

If you do want to go with the string union, we could define it somewhere as `HookType` or something. It's hard-coded in `usePlanHookInstanceFormState` currently.

Reply:
Good point it should be isPlayBookHook for a bolean returned type. 
Actually in the following PR#617 I changed it for string union, so I change it here.